README.md

```diff
@@ -157,10 +157,44 @@ To view your coverage report:
 
 <img width="500" src="screen.png">
 
-you can use any reporters that are supported by istanbul:
+you can use any reporters that are supported by istanbul and even use several at once:
 
 ```bash
-nyc report --reporter=lcov
+nyc report --reporter=<format> --reporter=<another format>
+```
+<format> is one of:
```

Comment:
This `format` should probably be escaped here, e.g. `` `format` ``? Otherwise it doesn’t display in the rendered README. :)